lib/Ast.ml

```diff
@@ -11,19 +11,8 @@ type location =
 (** Delimited locations *)
 type location_span = {begin_loc: location; end_loc: location}
 
-(** Origin blocks, to keep track of where variables are declared *)
```

Comment:
There's lingering doc on line 94 that refers to origin blocks, but there are no more origin blocks.  This is, by the way, what I meant about it being hard to keep the doc up to date in these cases and why I prefer not to have doc that repeats what the code's doing.

Reply:
Right. That makes sense. I'll remove it.